pkg/signal/signal_linux.go

```diff
@@ -125,3 +126,14 @@ func StopCatch(sigc chan os.Signal) {
 	signal.Stop(sigc)
 	close(sigc)
 }
+
+// SignalToString converts a given signal to its string
+// representation
+func SignalToString(sig syscall.Signal) (string, error) {
```

Comment:
```suggestion
func ToString(sig syscall.Signal) (string, error) {
```

signal.ToString()  is better than the stutter of single.SignalToString()